module/zfs/vdev.c

```diff
@@ -3678,7 +3678,9 @@ EXPORT_SYMBOL(vdev_offline);
 EXPORT_SYMBOL(vdev_clear);
 
 module_param(metaslabs_per_vdev, int, 0644);
+/* BEGIN CSTYLED */
```

Comment:
Warnings shows only on fuctions with 2+ lines with tabs before them (3+ run function code).

I'll do as you wrote.